extensions/src/main/java/org/sunbird/extension/user/UserExtension.java

```diff
@@ -0,0 +1,17 @@
+package org.sunbird.extension.user;
+
+import java.util.Map;
+
+/**
+ * This interface will provide utility methods to extend user management flows
```

Comment:
Change comment:
UserExtension is the interface for all extensions of Sunbird user profile. It defines methods for performing CRUD operations on user profile.